onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -62,6 +62,15 @@ struct KernelRegistryAndStatus {
 };
 }  // namespace
 namespace onnxruntime {
+
+class TestingInferenceSession : public InferenceSession {
+public:
+  using InferenceSession::InferenceSession;
+  const Model& GetModel() {
+    return *model_;
+  }
+};
+
```

Comment:
Please use InferenceSessionWrapper

Reply:
Done, thanks for the pointer (didn't realize this existed). We need access to the mutable graph though (in order to call `.resolve()` which clears out the inferred shapes) so I added an additional `GetMutableGraph` method to `InferenceSessionWrapper`.